quickannotator/db/__init__.py

```diff
@@ -87,6 +87,7 @@ class Tile(db.Model):
     # columns
     geom = Column(Geometry('POLYGON'))
```

Comment:
this GEOM can be removed, but you're missing the addition of a new column here called tile_id, which is a *relative* tile_id per image (as computed mentioned in the previous review). this field should be indexed jointly with imageid as a key (imageid + tileid). this field should also be the one stored in the annotation table and not "id", which is global. this saves a lot of querying/joining, as using the equations in the previous review, one can *always* quickly figure out what the tile_id of a particular annotation is, without having to rely on the Tiles table. do you see what i mean?